src/receive_imf.rs

```diff
@@ -3421,7 +3425,30 @@ async fn apply_mailinglist_changes(
     Ok(())
 }
 
-async fn apply_broadcast_changes(
+async fn apply_out_broadcast_changes(
+    context: &Context,
+    mime_parser: &MimeMessage,
+    chat: &mut Chat,
+    from_id: ContactId,
+) -> Result<GroupChangesInfo> {
+    ensure!(chat.typ == Chattype::OutBroadcast);
+
+    if let Some(_removed_addr) = mime_parser.get_header(HeaderDef::ChatGroupMemberRemoved) {
+        // The sender of the message left the broadcast channel
+        remove_from_chat_contacts_table(context, chat.id, from_id).await?;
```

Comment:
I think we should use the same "consistency algorithm" as we do for group members and names, to avoid applying changes from outdated messages. But for the first version it's fine to skip this

Reply:
I'm writing a list of small missing channel features, where I'll add this